### PR TITLE
Fix #182 in v2 branch

### LINKED ISSUE #182: localStorage throws, if saving data is disabled in browser settings --> app is broken
Please wrap localStorage access in try-catch. Same problem as in https://github.com/meteor/meteor/issues/5759

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Adding back a fix I posted a while ago.

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v2.0.1
+
+* Wrap localStorage in try/catch to avoid crash when disabled. Port fix from v1 branch. #182
```
Comment (mitar) [inline History.md]: I think `#182` should be a link here. So wrap it into a link.

Comment (StorytellerCZ) [reply History.md]: Fixed

**Comment by mitar** [comment]
BTW, I suggest you make all branches inside this repository, when making PRs. Also, always do a feature branch and do not name your branches the same as the target branch. Otherwise it is tricky if something does not get merged, then you have dirty your branch in your fork.

**Comment by mitar** [comment]
And thanks, published.

**Pull request merged**